tools/nvqpp/nvq++.in

```diff
@@ -39,10 +39,19 @@ function add_pass_to_pipeline {
 }
 
 function list_targets {
-	ls -I *.cpp -1 ${install_dir}/targets/ | sed -e 's/\.config$//'
+	ls -I *.cpp -1 ${install_dir}/targets/ | grep ".config$" | sed -e 's/\.config$//'
 	exit 0
 }
 
+function list_simulators {
+	for file in $(grep -L "PLATFORM_QPU=" $(ls ${install_dir}/targets/*.config));
```

Comment:
Should this be looking for LIBRARY_MODE_EXECUTION_MANAGER like the python code above?

Reply:
It's checked in the next line. (There's only one file as of now.)

Reply:
So it is.

This is a bit confusing though with the one grep in the `for` command and the other in the `do` command. I was tricked thinking they'd be at the same level like above.

Reply:
ahh.. I'm filtering non-matching files with the condition that appears more frequently and then skipping the one file. Using 'and' seemed inefficient to me, but, I can replace it for readability, if preferable.